src/Commands/Admin.py

```diff
@@ -1,10 +1,12 @@
 from CheckForUpdates import updateBot
 from EmbedHelper import AdminEmbed, ErrorEmbed, QueueUpdateEmbed
 from typing import List
-from bot import __version__
+from Types import Team
+from bot import __version__, LB_CHANNEL
```

Comment:
In `SixMans.py` the report function takes `LB_CHANNEL` as a parameter. Ideally we would want these two functions to be as similar as possible. Any particular reason you chose to import it instead?

Reply:
SixMans.py gets that parameter from bot.py, which is where I imported it from. In this method for forceReport, we bypass calling SixMans.report() solely because it's going to call reportMatch in Leaderboard.py without a parameter to know if the admin is forcing the report. Going straight to Leaderboard.reportMatch() with the adminOverride parameter makes it easy to force the report. Also, we needed slightly different embeds than those in SixMans.report()

Reply:
I don't think I did a good job asking my question as that wasn't what I was referring to. If you look at the implementation of `SixMans.report` one of the parameters it takes is `lbChannel`. In bot,py` where we call `SixMans.report` we pass `LB_CHANNEL` into that function as the `lbChannel` parameter.

My question is why not do the same here for `Admin.forceReport`? Why import it instead of passing it in as a parameter like `SixMans.report`?

Reply:
We don't need `LB_CHANNEL` for forceReport, we need it for a function that is called inside of it.

Reply:
> I don't think I did a good job asking my question as that wasn't what I was referring to. If you look at the implementation of `SixMans.report` one of the parameters it takes is `lbChannel`. In `bot.py` where we call `SixMans.report` we pass `LB_CHANNEL` into that function as the `lbChannel` parameter.
> 
> My question is why not do the same here for `Admin.forceReport`? Why import it instead of passing it in as a parameter like `SixMans.report`?

Looking at it now, I don't think that I had typed that up. VS Code might have automatically imported it as I was typing it out. Easy fix, I can quickly change it to take it as a parameter.

Reply:
Be sure to remove the import since it's no longer being used.